src/lib.rs

```diff
@@ -109,6 +109,7 @@
        html_root_url = "https://docs.rs/uuid")]
 #![deny(warnings)]
 #![cfg_attr(not(feature = "std"), no_std)]
+#![feature(const_fn)]
```

Comment:
Features are nightly only

Reply:
In order to make Uuid bytes private,So I add this const_fn featrue. Is there any other ways to create const Uuid？